### ISSUE TITLE
Document how to post a work to Po.et

### ISSUE THREAD

**Issue by lautarodragan** [poetapp/poet#303]
In a language-agnostic way

**Comment by pmosse** [comment]
Hey @lautarodragan! You mean the steps to build a code to send a work through the API?

**Comment by lautarodragan** [comment]
Yes. Since you're working on the new endpoint, you can document it after you're finished.

**Comment by lautarodragan** [comment]
Re-reading your question, think I got it wrong. I don't mean how to _code_ it, just the steps in English. What the API expects, what responses it will give, what happens afterwards, what it does, etc.
